core/templates/dev/head/pages/exploration_editor/EditorFirstTimeEventsService.js

```diff
@@ -0,0 +1,98 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ *  @fileoverview A Service registering analytics events for the editor
+ *  for events which are  only logged when they happen after the editor
+ *  is opened for the first time for an exploration.
+ */
```

Comment:
@aks681, one quick question, is it two spaces between the `*` and the first word in the lines 16 to 18 ? If yes, then please make it one space.

Reply:
ya, it was two spaces, fixed that